app/src/main/java/net/squanchy/service/proximity/injection/ProximityModule.java

```diff
@@ -14,17 +14,17 @@
 @Module
 public class ProximityModule {
 
-    private final Context context;
+    private final Application application;
 
-    public ProximityModule(Context context) {
-        this.context = context;
+    public ProximityModule(Application application) {
```

Comment:
Do you need to depend on the application? Also as far as I remember there's an `ApplicationContextModule` so you can just expose your application context as a dependency